frontend/src/services/eventHandlers/gameHandler.js

```diff
@@ -0,0 +1,13 @@
+import { EventBus } from "@/eventBus.js";
+import { Event } from "@/events";
+
+// Event handler for Game API
+export default class GameHandler {
```

Comment:
is game handler like the game master? or game controller?

Reply:
Game master? I think its to handle receive events